CONTRIBUTING.md

```diff
@@ -1,61 +1,130 @@
-# Contributing to Aya
+# Contributing Guide
 
-Thanks for your help improving the project!
+* [New Contributor Guide](#contributing-guide)
```

Comment:
I'm not a native English speaker, but is using capital letters everywhere necessary? I'd rather go with "Contributing guide", "Ways to contribute", "Find an issue" etc.

Reply:
It's [Title Case](https://en.wikipedia.org/wiki/Title_case)